e2e/cypress/integration/messaging/message_spec.js

```diff
@@ -182,4 +182,27 @@ describe('Message', () => {
         // * Focus to remain in the RHS text box
         cy.get('#reply_textbox').should('be.focused');
     });
+
+    it('MM-T336 Image thumbnail - expanded RHS', () => {
+        const filename = 'huge-image.jpg';
+
+        // # Post an image in center channel
+        cy.get('#centerChannelFooter').find('#fileUploadInput').attachFile(filename);
+        cy.contains(filename).should('be.visible');
+        cy.postMessage('{enter}');
+
+        // # Click reply arrow to open the reply thread in RHS
+        cy.clickPostCommentIcon();
+
+        cy.get('#rhsContainer').within(() => {
+            // # Observe image thumbnail displays the same
+            cy.get('img[src*="/preview"]').should('be.visible');
```

Comment:
This is missing verification for "observe image thumbnail displays the same (not cropped / cut off; can be smaller)".  Open to suggestion but one option could be by checking the aspect ratio of the rendered `img` element. Can be like:
```diff
const filename = 'huge-image.jpg';
+ const originalWidth = 1920;
+ const originalHeight = 1280;
+ const aspectRatio = originalWidth / originalHeight;
...

+ cy.get('img[src*="/preview"]').should('be.visible').and((img) => {
+    expect(img.width() / img.height()).to.be.closeTo(aspectRatio, 0.01);
+});
```